### PR TITLE
DH2

### PR DESCRIPTION
**Pull request by mbisan**
This evaluation is for a master's thesis.

### CODE REVIEW

**Comment by hermannsblum** [comment]
A new container image was submitted.

**Comment by hermannsblum** [comment]
@mbisan there were some errors with the CI script for your PR but they should be fixed now. The new error shows you what is wrong with your container. Please fix it by uploading a new container and re-triggering the CI script thorugh small cahnges.

**Comment by mbisan** [comment]
Hi, thanks for your quick reply, is it ok if I download the model's weights as part of the runtime or should the weights be included in the container?

**Comment by hermannsblum** [comment]
The container needs to be self-contained, so please include the weights in the container.

**Comment by hermannsblum** [comment]
A new container image was submitted.

**Comment by mbisan** [comment]
I uploaded the new container with the model preloaded.

**Comment by hermannsblum** [comment]
to trigger the CI, you need to commit a small change, eg adding a space somewhere in the json

**Comment by mbisan** [comment]
Hi, what are the segmentation outputs that must be saved?

I'm just saving the id of the class with the maximum logit (probability), and it saves an int64 array.

That is:

segmentation = logits.max(1)[1]

**Comment by hermannsblum** [comment]
A new container image was submitted.

**Comment by mbisan** [comment]
Everything is working properly, when should I expect results?

Thank you for your great work!

**Comment by hermannsblum** [comment]
results are online
